src/styles/helpers/container.scss

```diff
@@ -0,0 +1,14 @@
+.container {
+  padding-inline: 20px;
+  margin-inline: auto;
+
+  @include onTablet() {
+    padding-inline: 72px;
+    max-width: 600px;
+  }
+
+  @include onDesktop() {
+    max-width: 1020px;
+    padding-inline: 120px;
+  }
+}
```

Comment:
- The container max-width doesn't match Figma

Reply:
The container has max width of content. So in Figma (and video) we use max-width: 1020px to restrict the content.